Parse/PFObject.m

```diff
@@ -1634,7 +1634,9 @@ + (instancetype)objectWithClassName:(NSString *)className {
 
 + (instancetype)objectWithClassName:(NSString *)className dictionary:(NSDictionary *)dictionary {
     PFObject *object = [self objectWithClassName:className];
-    [object _mergeAfterSaveWithResult:dictionary decoder:[PFDecoder objectDecoder]];
+    [dictionary enumerateKeysAndObjectsUsingBlock:^(id key, id obj, BOOL *stop) {
+        object[key] = [[PFDecoder objectDecoder] decodeObject:obj];
```

Comment:
+1 nitpicking is important, shouldn't have missed that, my bad.  Didn't think it should be __block since we're not modifying the var.  Thanks for the quick feedback.